lib/evaluate/variable.h

```diff
@@ -276,7 +282,7 @@ template<typename A> class Designator {
 
   std::ostream &Dump(std::ostream &o) const {
     std::visit(common::visitors{[&](const Symbol *sym) {
-                                  o << sym->name().ToString();
+                                  o << GetSymbolName(sym).ToString();
```

Comment:
I'm starting to wonder if having a conversion operator std::string for CharBlock would be a nice thing.

Reply:
Go for it, see whether it would be a net improvement to readability and maintainability.